cloudslang_dsl_reference.rst

```diff
@@ -2100,6 +2100,8 @@ The sensitivity of an `input <#inputs>`__ or `output <#outputs>`__ is
 transitive, and is therefore determined by its ``sensitive`` property and by the
 sensitivity of variables used in its related value expression.
 
+The "sensitive" property works for both <#flow>__ and <#step>__ `inputs <#inputs>`__.
```

Comment:
the references don't work - syntax is probably not good - you can check either by biulding locally or on readTheDocs: http://cloudslang-docs.readthedocs.io/en/step_description/cloudslang_dsl_reference.html